### DISCUSSION TITLE
NASA’s Webb Identifies the Earliest Strands of the Cosmic Web

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#86] [Astrophysics and Physics]
Another surprise from the WST.

"A filament of 10 galaxies seen just 830 million years after the birth of the universe"  
[https://webbtelescope.org/contents/news-releases/2023/news-2023-124.html](https://webbtelescope.org/contents/news-releases/2023/news-2023-124.html)

“I was surprised by how long and how narrow this filament is,” said team member Xiaohui Fan of the University of Arizona in Tucson. “I expected to find something, but I didn't expect such a long, distinctly thin structure.”

**Comment by khuramonline** [comment]
Cosmic web is visible even beyond big bang. 

[The Cosmic Infrared Background](https://www.cfa.harvard.edu/news/cosmic-infrared-background) (Harvard University Website article dated 08–09–12)

> After meticulously accounting for known stars, diffuse emission, and known galaxies, the scientists found large structural features (fluctuations) consistent with their having been produced by the first generation of stars dating from an epoch more than twelve billion years ago. These early stars, at least according to the best current models of cosmic evolution, were not uniformly distributed across space. They formed as part of stupendous, **filamentary structures that evolved and grew from the tiny ripples in hot gas that pervaded the universe soon after the big bang.** The results of this new paper lend support both to the current theory of the large scale structure of the universe and to our ideas about how the first generation stars formed.

So by 1912, the idea about CIB was that large portion of CIB do not come from visible sources but come from **"early stars" in filamentary structures.**

Later on (by 2016) it was found that CIB is not fully accounted for by proto galactic sources.

[The Cosmic Infrared Background | Highlights of Astronomy | Cambridge Core](https://www.cambridge.org/core/journals/highlights-of-astronomy/article/the-cosmic-infrared-background/7259ECEB88B6B64621248F0F03AB5281)

> DIRBE measurements do not confirm some previous data from rockets and IRAS. Preliminary upper limits on the cosmic infrared background are generally above theoretical expectations for pregalactic and protogalactic sources, and substantially above estimated lower limits based upon observations of external galaxies. Careful foreground modeling is required to reduce these limits or identify an isotropic residual.

And now JWST has actually shown the infrared filamentary structures. These structures are so big that they cover the 100% of the visible sky. And These structures are located beyond -- far far beyond the official big bang.

In following screenshot, I have highlighted only two filamentary structures.
 
![image](https://github.com/a-cosmology-group/acg/assets/132253587/b8fff0e9-8b4a-4fc5-a764-9e60ab74b52a)

**Comment by khuramonline** [comment]
There are two outstanding questions for **believers** of static and infinite universe.

First question is that if universe is static and infinite then where is the *** 100% bright sky?

Second question is that if JWST is the most powerful infrared telescope then why so far JWST has achived nothing about CIB? Why the filamentary sources of CIB have not been identified by the JWST so far?

**Reply by khuramonline** [reply]
There is also a third question ---- if it is static and infinite universe then why the maximum distant visible object is located (official distance) at 13.5 billion light years. **Why is there a dark age beyond that distance?**

**Reply by HanDeBruijn** [reply]
> First question is that if universe is static and infinite then where is the *** 100% bright sky?

> There is also a third question ---- if it is static and infinite universe then why the maximum distant visible object is located (official distance) at 13.5 billion light years. __Why is there a dark age beyond that distance?__

That's a bit of a complicated math story, but both questions can be managed theoretically: read the [__G. de Vaucouleurs__](http://www.alternatievewiskunde.nl/cosmology/helaas.htm) section at my website.

**Reply by khuramonline** [reply]
Ok. You got a task to manage. Hope for the best.

**Comment by khuramonline** [comment]
All the images of Cosmic web are just projections. Only one category of images present the trure picture of cosmic web. Following is just one example. This is image of one of the so-called farthest visible galaxies (glass z-13). Cosmic web is actually visible in the background of glass z-13

![image](https://github.com/a-cosmology-group/acg/assets/132253587/392676b0-700a-42fb-a79b-943cd668e168)

This web is everywhere --- like a wall on sky in every direction. No need to see only glass type galaxies.
